contrib/bentoml/OWNERS

```diff
@@ -0,0 +1,3 @@
+approvers:
+  - yetone
+  - yubozhao
```

Comment:
```suggestion
  - yubozhao
  - juliusvonkohout
```

Reply:
@yetone could you please approve this suggestion? cc: @juliusvonkohout, @EricBentoML

Reply:
Yes if you keep yubozhao he must be member of Kubeflow as well. If you do not have any members in your company use me and @kimwnasptd

Reply:
Hello Julius. 

I am part of the release 1.7 team and I should already be part of the kubeflow member.